app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -128,6 +137,15 @@ class EventDetailsFragment : Fragment() {
                 showEventErrorScreen(true)
             })
 
+        eventViewModel.eventFeedback.observe(viewLifecycleOwner, Observer {
+            feedbackAdapter.addAll(it)
+            if (it.isEmpty()) {
+                rootView.feedbackContainer.visibility = View.GONE
```

Comment:
Replace with `rootView.feedbackContainer.isVisible = false`

Reply:
> Replace with `rootView.feedbackContainer.isVisible = false`

What is the advantage?

Reply:
Much clearer and using booleans (primitives)

Reply:
@liveHarshit any other change ?